department-of-veterans-affairs/caseflow
# remove featuretoggle useAMAActivationDate

Resolves [Appeals-4974](https://jira.devops.va.gov/browse/JIRA-4974)

# Description
Removes - use AmaActivationDate Feature toggle. 
## Acceptance Criteria
- [ ] All rspec test should pass
- [ ] if receipt date is old than 2019-02-19 then it should prompt error message during intake process.

## Testing Plan

1. Login as BVAIntake user
2. Try intaking 10182 form
3.  set value of Receipt date to be 01/01/2019
4. Validation error saying -" Receipt date cannot be prior to 02/19/2019" should be visible. when you click on 'Continue to next step'
5. try step 1 to 4 for HRL and SC 
6. try changing the claimant to 'Claimant not listed' and this error should still be present. 
 


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
